data/BGC0001538.json

```diff
@@ -1,49 +0,0 @@
-{
```

Comment:
This change should be in the previous commit

Reply:
Fixed